script/release

```diff
@@ -595,7 +595,7 @@ unless( $opts{k} // $release->config->skip_kwalitee  ) {
 	$release->clean;
 	$release->build_makefile;
 	$release->make;
-	$release->dist;
+	$release->dist unless $skip_dist;
```

Comment:
The `check_kwalitee` needs the local dist file to exist. Are you seeing something different? But, I would think if you are testing against 400 perls, you don't want to test kwalitee 400 times either, so you'd have `release -k -t`.